client/react/components/chat/chatView.jsx

```diff
@@ -0,0 +1,36 @@
+import { Button, Comment, Container, Menu, Icon } from 'semantic-ui-react';
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+import { push } from 'react-router-redux';
+
+
+import './styles.css';
+
+const ListComments = ({
+    chatLog
+}) => {
+        return chatLog.map((message,i) => (
+        <Comment key= {i}>
+            <Comment.Content>
+                <Comment.Author as='a'>{message.userName}</Comment.Author>
```

Comment:
We should implement a 'go to profile'.

Something like a link to /profile/{id}